indra/newview/llxmlrpctransaction.cpp

```diff
@@ -104,6 +100,8 @@ class LLXMLRPCTransaction::Impl
 
     std::string         mResponseText;
     LLSD                mResponseData;
+	bool                mHasResponse;
+	bool                mResponseParsed;
```

Comment:
```suggestion
    bool                mHasResponse;
    bool                mResponseParsed;
```

Please fix this to make pre-commit happy, and then I'll gladly accept the change.
```
Check Indentation (Spaces)...............................................Failed
- hook id: indent-with-spaces
- exit code: 1

indra/newview/llxmlrpctransaction.cpp: 2 lines starting with tabs found
```

Reply:
Here you go...

Reply:
Thank you very much!